web-extensions/chrome/entrypoint.js

```diff
@@ -41,9 +41,9 @@ if (window.top === window) {
         const engine = new BmcEngine(window.location.origin,
                                      window.location.hostname,
                                      response.resource.comic);
-        console.log('Instanciated BmcEngine');
+        LOGS.log('S42');
         engine.setup();
     });
 } else {
-    console.warn(`Attempting to reload for iframe: ${window.location}`);
+    LOGS.warn('E0003', {'location': window.location});
```

Comment:
the string is still expecting `iframe` in its format :)

Reply:
Woups. Fixed!